### DISCUSSION TITLE
ATSAS-4.0.1-1

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#1947] [Announcements]
# ATSAS-4.0.1-1

> [!IMPORTANT]
> As of ATSAS-4.0.0, every installation of ATSAS requires a personalised license file! Users of ATSAS that are affiliated with a research or educational institute can request an academic license free of charge over at [BIOSAXS GmbH](https://www.biosaxs.com/download). Otherwise, please contact [info@biosaxs.com](mailto:info@biosaxs.com) to request an offer for a commercial license.

 - [What's new in 4.0.1](https://biosaxs-com.github.io/atsas/4.0.1/)
 - [Installation Instructions](https://biosaxs-com.github.io/atsas/4.0.1/install/)
 - [Manuals](https://biosaxs-com.github.io/atsas/4.0.1/manuals/)
 - [Discussions](https://github.com/biosaxs-com/atsas-community/discussions)
 - [Issue Reports](https://github.com/biosaxs-com/atsas-community/issues)
 - [Wiki](https://github.com/biosaxs-com/atsas-community/wiki)

<hr /><em>This discussion was created from the release <a href='https://github.com/biosaxs-com/atsas-community/releases/tag/atsas-4.0.1-1'>ATSAS-4.0.1-1</a>.</em>

**Comment by LewisAJackson** [comment]
Hi, I am trying to install the saspy PyMOL plugin but I can't find the saspy file in the ATSAS-4.0.1-1 folder, and it doesn't seem to work when I download the saspy file from GitHub. Any help thanks

**Reply by franke-biosaxs** [reply]
SASpy is not part of the ATSAS package any more as the latest Gtk-based version it is not compatible with Qt-based PyMol. As you have noticed. Someone would need to reimplement and update it entirely from scratch with Qt-based widgets and update the calls to the tools as needed.

As the original implementation is Open Source, anyone can do it, but there have been no volunteers in the last four years. However, if someone would like to sponsor this activity for the community, please contact [BIOSAXS GmbH](https://biosaxs.com/) for an offer.

**Comment by LewisAJackson** [comment]
Hi,

Thanks for the reply

Do you know what else I would be able to use to fit my AlphaFold.PDB structure into the SAXS data

Thanks for the help,
Lewis

From: Daniel Franke ***@***.***>
Date: Sunday, 29 September 2024 at 14:31
To: biosaxs-com/atsas-community ***@***.***>
Cc: JACKSON, LEWIS (PGR) ***@***.***>, Comment ***@***.***>
Subject: Re: [biosaxs-com/atsas-community] ATSAS-4.0.1-1 (Discussion #1947)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

SASpy is not part of the ATSAS package any more as the latest Gtk-based version it is not compatible with Qt-based PyMol. As you have noticed. Someone would need to reimplement and update it entirely from scratch with Qt-based widgets and update the calls to the tools as needed.

As the original implementation is Open Source, anyone can do it, but there have been no volunteers in the last four years. However, if someone would like to sponsor this activity for the community, please contact BIOSAXS GmbH<https://biosaxs.com/> for an offer.

**Comment by maxim-biosaxs** [comment]
If you want to compute the intensity fit from the AlphaFold model to the SAXS curve, please use CRYSOL.

In case the task is to superimpose AlphaFold model with the precomputed SAXS shape, CIFSUP is the option

**Comment by wpk390** [comment]
Hi,
I installed ATSAS-4.0.1-1 on my Windows PC. Launched ATSAS  Application Launcher -> Console. Then I ran autorg in the console. I got an error 

D:\HKEY_CURRENT_USER\Software\BIOSAXS GmbH>autorg
error: ATSAS license: no such file 'atsas.lic' in 'ATSAS=D:\Program Files\ATSAS-4.0.1 '

Could you please assist in diagnosing and resolving this issue? Let me know if any additional information is needed.

Looking forward to your response.

Thanks,
Dilip

**Comment by fmneto** [comment]
Trying to install ATSAS, but the site for requesting a license file is not available as of March 28, 2025.

**Reply by franke-biosaxs** [reply]
Seems to work fine here? We recently changed the service provider and there was a short interruption due to that. Maybe [try again](https://www.biosaxs.com/download.php)? If the problem persists, please provide more details and we try to look into it.
